src/bin/encode.rs

```diff
@@ -188,6 +188,9 @@ fn main() {
             Arg::with_name("statistics")
                 .long("show-stats")
                 .help("Show statistics."),
+            Arg::with_name("dump")
```

Comment:
Please don't undo that change.

Reply:
(we'll need to readd the `dump` option, but first we need to reimplement it :) )